sycl/include/CL/sycl/detail/context_impl.hpp

```diff
@@ -129,6 +129,9 @@ class context_impl {
 
   KernelProgramCache &getKernelProgramCache() const;
 
+  /// Returns true iff context contains the given device.
```

Comment:
```suggestion
  /// Returns true if context contains the given device.
```

Reply:
Thank you for suggestion, but I used iff intentionally here, 'iff' means 'if and only if'

Reply:
Does it really needed here?

Reply:
I suggest to use *if and only if* if we want the mathematics meaning (which seems the case here...).
I have the feeling that in programs there are a lot of *if* that should be *iff* but usuallly a source program is not a mathematical proof...

Reply:
Ok, I replaced 'iff' with 'if and only if'